include/drivers/driver_enums.hpp

```diff
@@ -9,6 +9,9 @@ enum class DriverCommand {
     BUTTONS = 2,
     TERMINAL_OUTPUT = 3,
     TIMER_CANCEL = 4,
+    SET_PIN = 5,
+    CLEAR_PIN = 6,
+    SET_PIN_OUTPUT = 7,
```

Comment:
`set_pin` and `set_pin_output` are kinda confusing

Reply:
cahnging to `config_pin_output`